main/src/ca/uwaterloo/flix/language/phase/Typer.scala

```diff
@@ -1601,17 +1601,20 @@ object Typer {
 
 
       case KindedAst.Expression.NewChannel(exp, elmType, loc) =>
+        val regionVar = Type.freshVar(Kind.Bool, loc, text = FallbackText("region"))
```

Comment:
This should be False.

Reply:
Sorry Magnus, what's the "this" that should be False? Are you suggesting that I shouldn't have a `regionVar` here at all and instead pass `Type.False` to `mkSender` and `mkReceiver` below. That contradicts what you said earlier, so I guess not? But then I don't know what else you might mean?